### DISCUSSION TITLE
Inexplicable "Not ready, Please check your connection"

### DISCUSSION THREAD

**Discussion by almendesb** [rustdesk/rustdesk#2582] [Q&A]
I have a RustDesk server running for about a month, with which I make remote access possible between approximately 30 computers.

Yesterday, all of a sudden, one of these computers simply cannot access my RustDesk server, on the client the message "Not ready, Please check your connection" appears. However, this machine is normally connected to the internet and apparently there has been no changes in the environment. The other machines, in other networks environments and in others locations, continue to access my RustDesk server as normal.

To make the situation even stranger, on that same machine where the message "Not ready, Please check your connection" appears, if I remove data from my RustDesk server, allowing the Client to access the general RustDesk server, the message appears "Ready" indicating that the computer is normally accessing external networks.

Could someone help me identify what the problem is in this case? Are there any cases where the RustDesk server blacklists IPs automatically? Are there any Whitelists on the RustDesk Server?

**Comment by working-name** [comment]
Similar issue here but isolated to a single computer. All the others are ok, self-hosted ID+relay server is up, ports are open, IPs are not blocked/filtered. The computer is using a router as a wifi client to give it a hardwired connection but it's otherwise normal. Other remote desktop software works.

**Comment by almendesb** [comment]
To make things even more inexplicable: I formatted my Ubuntu server, reinstalled RustDesk, replaced all public keys on all computers... And As a result: Everyone continues to access normally, but the one that was not connecting keeps showing the same message "Not ready, Please check your connection" . However, remaining the same, when I remove data from my own server in the RustDesk Client and the RustDesk Client tries to access the general public server, it connects normally.

**Comment by rustdesk** [comment]
#4671

**Comment by dw1161** [comment]
Same problem here but I just fixed it magically.  I was hosting it inside a docker on synology ds920+ in one of my house, and clients are from another house using windows 11 and macos, after normally working for several days, both clients started showing "not ready and check connection". Also I noticed that by connecting to iphone's hotspot instead of home wifi or lan these two clients worked okay, and from my workplace's win11 laptop it works fine too(using workplace wifi). After trying 3 days(tried reinstall and firewall and all other possiblilities), I just magically fixed the problem by remote ssh to synology's hbbs container( should be under /volume1/@docker/containers/xxxxx) and print out each of all the files' content under the hbbs container's path,  including the "log.db" file, use command like "tail -f log.db". After that, both windows and macos client returned back to green light again. Not really know what just happened...Seems stucked on something and after printing it went through...

**Comment by joseluis4000** [comment]
En mi caso, no tengo servidor configurado
Y el problema surgió después de realizar una actualización y pese a que intenten regresar a la versión anterior no fue posible volver a conectarme, por esta razón no he actualizado las demás PC a las que me conecto.
Ya intenté registrar como nueva regla la apertura de los puertos (firewall) del 21115 al 21119, pero no obtuve ningún cambio

Espero que alguien haya podido solucionar?!?!?!? y comparta...

**Comment by raksalim** [comment]
Fix (for me):

Do you have `Enable direct IP access" on at port 21118`?

If so, try disabling it. You may be blocking the 'hbbs' container and forcing it to endlessly restart.
